src/Source/EntwinePointTileSource.js

```diff
@@ -40,12 +40,17 @@ class EntwinePointTileSource extends Source {
             this.parse = metadata.dataType === 'laszip' ? LASParser.parse : PotreeBinParser.parse;
             this.extension = metadata.dataType === 'laszip' ? 'laz' : 'bin';
 
-            if (metadata.srs && metadata.srs.authority && metadata.srs.horizontal) {
-                this.crs = `${metadata.srs.authority}:${metadata.srs.horizontal}`;
-                if (!proj4.defs(this.crs)) {
-                    proj4.defs(this.crs, metadata.srs.wkt);
+            if (metadata.srs) {
+                if (metadata.srs.authority && metadata.srs.horizontal) {
+                    this.crs = `${metadata.srs.authority}:${metadata.srs.horizontal}`;
+                    if (!proj4.defs(this.crs)) {
+                        proj4.defs(this.crs, metadata.srs.wkt);
+                    }
+                } else if (metadata.srs.wkt) {
+                    proj4.defs('unknown', metadata.srs.wkt);
+                    this.crs = proj4.defs('unknown').name;
+                    proj4.defs(this.crs, proj4.defs('unknown'));
```

Comment:
Same here, the `wkt` parser breaks the type invariant that what type returned `by proj4.defs(crs)` (i.e. the [`ProjectionDefinition`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/72db0783554789ef424e5dc5a207a028c2a0169f/types/proj4/index.d.ts#L36) type) shall return a defined `title` property. We could use this property to uniquely define a projection.

I suggest that we could contribute to `proj4` such fix (using srs.authority) to define the `title` property.